operator/pkg/controller/karmada/validating.go

```diff
@@ -97,6 +98,40 @@ func validateETCD(etcd *operatorv1alpha1.Etcd, karmadaName string, fldPath *fiel
 	return errs
 }
 
+// validateCommonSettings validates the common settings of a component, including PDB configuration
+func validateCommonSettings(commonSettings *operatorv1alpha1.CommonSettings, fldPath *field.Path) (errs field.ErrorList) {
```

Comment:
```suggestion
func validatePDBSettings(pdbConfig *operatorv1alpha1.PodDisruptionBudgetConfig, fldPath *field.Path) (errs field.ErrorList) {
```
This function focuses on validating PDB. 

PS: You might need to update the comment and implementation accordingly.